app/javascript/containers/App.js

```diff
@@ -9,12 +11,13 @@ const App = props => {
   return(
     <Router history={browserHistory}>
       <Route path='/'>
```

Comment:
FYI since you don't have any component you want to show on every page (i.e. a NavBar), you do not need to nest this route! I would rather you remove lines 13 and 19, and change line 14 from:
```
<IndexRoute component={GenreListContainer}/>
```
to:
```
<Route path="/" component={ GenreListContainer } />
```